### DISCUSSION TITLE
osgeo-pyqt formula is missing

### DISCUSSION THREAD

**Discussion by slarosa** [OSGeo/homebrew-osgeo4mac#1403] [Q&A]
Hi all, I am trying to upgrade qt from homebrew and I am getting the following:

```
Error: could not get runtime dependencies from /usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/osgeo-pyqt-webkit.rb!
Error: No available formula with the name "osgeo-pyqt".
```
It seems that pyqt formula is not available. Any hint?

**Comment by alazarolop** [comment]
At this moment, no maintained formula from the tap is using it, so it was deprecated. The formula is now located in "boneyard" directory. Which of your formulas still depends on this one?

**Reply by slarosa** [reply]
I am getting that error while upgrading to qt-5.15.2 :
```
==> Summary
🍺  /usr/local/Cellar/qt/5.15.2: 10,688 files, 365.0MB
Error: could not get runtime dependencies from /usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/osgeo-pyqt-webkit.rb!
Error: No available formula with the name "osgeo-pyqt".
```

**Comment by alazarolop** [comment]
Because the formula depended on "qt". If you don't know what formula "osgeo-pyqt" is depending, I reckon you could remove it. You can try to add `--dry-run` tag e.g. `brew remove osgeo-pyqt --dry-run`.

**Reply by slarosa** [reply]
After removing osgeo-pyqt I get:
```
Error: could not get runtime dependencies from /usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/osgeo-qscintilla2.rb!
Error: No available formula with the name "osgeo-pyqt".
```
because osgeo-qscintilla2.rb depends on it.

**Comment by alazarolop** [comment]
Have you installed `osgeo-qgis` through the tap at any time?

**Reply by slarosa** [reply]
No, I have always compiled qgis via cmake using some osgeo formula.
